### DISCUSSION TITLE
官网打不开了吗

### DISCUSSION THREAD

**Discussion by Maoweihang1995** [didi/LogicFlow#2014] [General]
昨天想上官网查api，发现打不开了，今天也还是打不开

**Comment by EdwinOlders11** [comment]
我挂上梯子后就能打开了，难不成是被墙了？

**Reply by Maoweihang1995** [reply]
我挂梯子也打不开。。

**Comment by boyongjiong** [comment]
https://logic-flow.cn/   备用的这个试一下呢？

**Comment by boyongjiong** [comment]
所有网络都打不开吗？可以切换 WiFi 和 流量，试试看能不能打开

明年会做专项处理定位和解决这个问题，最近太忙了，我们通过大盘观察的话好像没有那么多打不开的情况，但根据反馈确实是存在打不开的问题的

**Reply by Maoweihang1995** [reply]
不用梯子能打开了
